README.md

```diff
@@ -14,6 +14,9 @@ This module helps Cirrus CI to parse artifacts and find relative annotations to
 * Ruby
   * RSpec
   * RuboCop
+* Swift
+  * XCLogParser
```

Comment:
Maybe also a link to https://github.com/spotify/XCLogParser

```suggestion
  * [XCLogParser](https://github.com/spotify/XCLogParser)
```

Reply:
See https://github.com/cirruslabs/cirrus-ci-annotations/pull/27/commits/14d3e8c1319e2567f81880a349bd9a4306583b87.